core/src/main/java/tc/oc/pgm/command/graph/MatchPlayerProvider.java

```diff
@@ -11,7 +11,7 @@
 import tc.oc.pgm.api.player.MatchPlayer;
 import tc.oc.pgm.util.text.TextException;
 
-final class MatchPlayerProvider implements BukkitProvider<MatchPlayer> {
+public final class MatchPlayerProvider implements BukkitProvider<MatchPlayer> {
```

Comment:
Unsure if this is supposed to be a part of the API in PGM? What case do you have for needing this?

Reply:
For Ingame, we're using `MatchPlayerProvider` and `MatchProvider` in the command framework. These classes need to be public because our `CommandModule` is not in the `tc.oc.pgm.command.graph` package.

Reply:
Yeah, I still think that these providers should all be made public as a part of the API, or you could make your own providers through `PGM.get()`. You can use bukkit `World's and/or `Entity`s to get the correct `Match` and `MatchPlayer`s.

Anyone else have any opinions on this?